static/src/js/components/OneOfField/OneOfField.jsx

```diff
@@ -98,12 +101,9 @@ class OneOfField extends React.Component {
       }
     }
 
-    /*
-    *  The code below commented out caused a bug where it would revert the option selected in some cases.
-    */
-    // if (newState !== this.state) {
-    //   this.setState(newState)
-    // }
+    if (newState !== this.state) {
+      this.setState(newState)
+    }
```

Comment:
Were you able to replicate the use cases referenced in the old comment? Is that bug resolved now?

Reply:
This was a bug in the prototype. By having this if statement commented was causing the bug the user reported. By uncommenting it fixed the issue